CHANGELOG.md

```diff
@@ -4,21 +4,21 @@
 
 The full documentation for MIVisionX is available at [https://rocm.docs.amd.com/projects/MIVisionX/en/latest/doxygen/html/index.html](https://rocm.docs.amd.com/projects/MIVisionX/en/latest/doxygen/html/index.html)
 
-##  (unreleased) MIVisionX 3.2.0
+## MIVisionX 3.2.0 for ROCm 7.0.0
 
 ### Changed
 
 * Setup: OpenCV package install for Ubuntu
 * AMD Clang is now the default CXX and C compiler
 * The version of OpenMP included in the ROCm LLVM project is now used instead of `libomp-dev/devel`
-* VX_RPP extension : Update to version 3.1.0
+* VX_RPP extension : Version 2.0.0 release
```

Comment:
@kiritigowda  vx_rpp version is 3.10, rpp version is 2.0.0. Please undo this

Reply:
https://github.com/ROCm/MIVisionX/blob/0394f4bb82710d41a7d4f49740df0dc354f8ffcf/amd_openvx_extensions/amd_rpp/CMakeLists.txt#L31 -- it is 2.0

Reply:
3.10 is the cmake version

Reply:
Not referring to the cmake version. I am referring to the vx_rpp_extension.h version: https://github.com/ROCm/MIVisionX/blob/develop/amd_openvx_extensions/amd_rpp/include/vx_ext_rpp_version.h.
The rocAL line is not a bug. it refers to this file

Reply:
Sorry about the typo, meant `3.1.0`

Reply:
I think the cmake `set(VERSION 2.0.0)` might have to be updated to match the header file